### ISSUE TITLE
Extension not installing in Mechanic

### ISSUE THREAD

**Issue by gferreira** [roboDocs/roboFabDocsExtension#1]
for some reason the extension is not installing in Mechanic:

![screen shot 2015-10-05 at 08 22 16](https://cloud.githubusercontent.com/assets/62717/10279232/38686356-6b3a-11e5-8e6e-595c022c6823.png)

…but it works normally if I double-click the extension file.

any idea what is going wrong? thanks

**Comment by typemytype** [comment]
I guess git sub module inside the lib folder doesn't worked well 




gr Frederik
www.typemytype.com

**Comment by gferreira** [comment]
yep, my bad. I've now generated the .roboFontExt file inside the roboFabDocs repository, to avoid this problem.

I think it should work now if the RoboFab Docs extension registry in Mechanic is updated to this:

![screen shot 2015-10-05 at 10 09 20](https://cloud.githubusercontent.com/assets/62717/10281236/76ee0068-6b49-11e5-8280-910f65620c3e.png)

and then we delete this separate repo (roboFabDocsExtension).

I hope it makes sense. sorry for the mess!
